cppcache/include/geode/Exception.hpp

```diff
@@ -19,6 +19,7 @@
 
 #ifndef GEODE_EXCEPTION_H_
 #define GEODE_EXCEPTION_H_
+#pragma warning(disable : 4275)
```

Comment:
What does this disable? Comment?

Reply:
Hmm. I am surprised, I didn't comment.... I will fix that.